notes-with-indexeddb/notes.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Notes</title>
+    <link rel="stylesheet" type="text/css" href="notes.css">
+  </head>
+
+<body>
+  <h1>Notes</h1>
+	
+	<form id="add-new-item">
```

Comment:
Please expand tabs to spaces throughout this file, and use consistent 2-space indentation.